includes/class-activitypub.php

```diff
@@ -703,6 +704,36 @@ public static function prevent_empty_post_meta( $check, $object_id, $meta_key, $
 		return $check;
 	}
 
+	/**
+	 * Get post meta.
+	 *
+	 * @param mixed  $meta_value The meta value.
+	 * @param int    $object_id  ID of the object metadata is for.
+	 * @param string $meta_key   Metadata key.
+	 *
+	 * @return mixed The meta value.
+	 */
+	public static function default_post_metadata( $meta_value, $object_id, $meta_key ) {
+		// Check if the meta key is `activitypub_content_visibility`.
+		if ( 'activitypub_content_visibility' !== $meta_key ) {
+			return $meta_value;
+		}
+
+		// If the post is federated, return the default visibility.
+		$is_federated = \get_post_meta( $object_id, 'activitypub_status', true );
+		if ( 'federated' === $is_federated ) {
```

Comment:
```suggestion
		if ( 'federated' === \get_post_meta( $object_id, 'activitypub_status', true ) ) {
```
`$is_federated` is not used again.